Сетка страниц и крупных блоков на флексах

Dar-Suvorov opened the pull request:
4.22. Базовая стилизация всех страниц 

---
:mortar_board: [Сетка страниц и крупных блоков на флексах](https://up.htmlacademy.ru/htmlcss/37/user/2344675/tasks/10)

Dar-Suvorov committed:
исправляет баги

По результатам созвона в сб 17.06:

Изменения:
- CSS: исправлено несоответствие критерию CSS-02
- страница imdex, блок Hero: изменен способ центрирования надписи
- CSS: улучшен нейминг классов
- исправлены мелкие баги

- страница catalog: звездочки пока оставил контентными изображениями

Вопросы в комментариях:
- CSS, стр. 142, 154, 217

Dar-Suvorov committed:
fix

Добавляет точки в alt-текст

Dar-Suvorov committed:
стилизует кнопки страницы index

Dar-Suvorov committed:
Стилизует кнопки страницы catalog

Dar-Suvorov committed:
убирает неактуальные комментарии

Dar-Suvorov committed:
стилизует шапку

Сетка шапки на флексах.

Dar-Suvorov committed:
стилизует подвал

Dar-Suvorov committed:
стилизует страницу imdex

Кроме формы подписки

aimenin commented on test-index.css:
```diff
@@ -0,0 +1,30 @@
+span {
```
не пройдет по критерию
![image](https://github.com/htmlacademy-htmlcss/2344675-sedona-37/assets/49648795/6299877c-c000-47ea-a5d3-e0ed0a216109)

Лучше задать span класс и в классе стилизовать его

Dar-Suvorov replied:
Это файлы для тестирования стики футера. Уберу из проекта, когда он получится.

aimenin commented on test-index.css:
```diff
@@ -0,0 +1,30 @@
+html {height: 100%;}
+
+body {
+	display: flex;
```
поехало выравнивание слегка

aimenin commented on css/styles.css:
```diff
@@ -59,129 +40,447 @@ a {
+}
+
+.white-on-blue .main-advantage-line {
+  border: 2px solid #ffffff3D;
```
цвет поплыл, мне кажется

Dar-Suvorov replied:
Привел все к единому стилю через rgba(). Перед сдачей перепроверю еще раз.

aimenin reviewed (commented):
Впринципе все хорошо, обсудим детали на встрече

Dar-Suvorov committed:
fix: по результатам созвона

Исправления по результатам созвона 28.06.

Dar-Suvorov committed:
Меняет способ стилизации блока "Жилье ..."

Удаляет стилизацию через nth-of-type, включает стилизацию через дополнительные классы.

Dar-Suvorov committed:
catalog - переносит шапку

Копирует готовую шапку со страницы index.

Dar-Suvorov committed:
catalog - стилизует блок фильтров

кроме контролов и части с диапазоном цен

Dar-Suvorov committed:
catalog - стилизует блок выдачи

Кроме стилизации самих карточек.

Dar-Suvorov committed:
catalog: стилизует блок подписки

Кроме самой формы подписки.

aimenin commented on catalog.html:
```diff
@@ -119,25 +122,28 @@ <h2 class="visually-hidden">Подбор гостиницы в Седоне</h2>
+              <h2 class="visually-hidden">Гостиницы в Седоне</h2>
+              <p class="catalog-cards-caption-title">Найдено гостиниц: <span>38</span>
+              </p>
+              <div> <!-- Можно оставить div без классов? Он тут нужен только для группировки кнопок в единый флекс-элемент. -->
```
да, данная обертка лишней не будет, так как позволяет сгруппировать элементы

aimenin reviewed (commented):
Мне все нравится
